tests/integration.rs

```diff
@@ -2176,7 +2181,7 @@ mod tests {
                     "--net",
                     guest.default_net_string().as_str(),
                     "tap=,mac=8a:6b:6f:5a:de:ac,ip=192.168.3.1,mask=255.255.255.0",
-                    "tap=,mac=fe:1f:9e:e1:60:f2,ip=192.168.4.1,mask=255.255.255.0",
+                    "tap=mytap1,mac=fe:1f:9e:e1:60:f2,ip=192.168.4.1,mask=255.255.255.0",
```

Comment:
I'm not following here, if `mytap1` does not exist, shouldn't you add some check to verify that this is unusable from the VM?

Reply:
In fact this shows up we *never* check the tap device names on the host side.

Reply:
> I'm not following here, if `mytap1` does not exist, shouldn't you add some check to verify that this is unusable from the VM?

The test is for named tap that doesn't exist. i.e. Please create me a new tap device with this particular name rather than with a generic one.